static/css/variables.less

```diff
@@ -17,6 +17,10 @@
 @button-text-color: #333333;
 @button-text-inverse-color: #DDDDDD;
 @deceased-color: #A5A8AA;
+@unverified-color: #A7A9AC;
+@verify-error-color: #E0381F;
+@verify-valid-color: #789A11;
+@verify-valid-icon-color: #8EB41D;
```

Comment:
I'm quite uncomfortable adding in custom colours - try and reuse the existing colours where possible to keep our palette consistent. For example, can we change:

- @unverified-color => @label-color (this has the added advantage that the options don't look disabled)
- @verify-error-color => @error-color
- @verify-valid-color => @ok-color
- @verify-icon-valid-color => @ok-color

Feel free to discuss this with design - I know they want to keep the palette consistent too.

Reply:
Design was already very specific with the colors that should be used - both in the issue itself and the design document. I assumed they were specific for a reason :)